cli/features/tutorial.feature

```diff
@@ -36,9 +36,9 @@ Feature: Following the tutorial
       description: A todo application
       version: 0.0.1
 
-      bus:
-        type: exocom
-        version: 0.21.7
+      dependencies:
+        - type: exocom
+          version: 0.21.7
```

Comment:
Something else to consider...if we eventually want to support configuring these app wide depenencies, should we format it like so:
```
dependencies:
  - type: exocom
      version: 0.21.7
  - type: mongo3.4.0
      version: 3.4.0
      ports:
        - '27017:27017'
      environment-variables:
         - EXOCOM_PORT: 3100
      volumes:
         - {some_mount..}
```

in which case we might even consider changing `type` to `name`
```
dependencies:
  - name: exocom
      version: 0.21.7
  - name: mongo3
      version: 3.4.0
      ports:
        - '27017:27017'
      environment-variables:
         - EXOCOM_PORT: 3100
      volumes:
         - {some_mount..}
```